### PR TITLE
cmd for draw images in framebuffer

### PR DESCRIPTION
**Pull request by FilippChubukov**

### CODE REVIEW

File: src/cmds/testing/fb_draw_image/fb_draw_image.c [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	printf(
+		"Usage : fb_draw_image [FILE]\n"
+		"If [FILE] format is supporting then draw it to the framebuffer\n"
+	 );
+}
+
+static int get_fbp(uint8_t *fbp, struct fb_info *fb_info, uint8_t *img, uint32_t width, uint32_t height) {
+	int x, y, pos = 0;
+	long int idx = fb_info->var.xoffset + fb_info->var.xres * fb_info->var.yoffset;
+	int bytes_per_pixel = fb_info->var.bits_per_pixel / 8; /* Figure out the size of the screen in bytes */
```
Comment (Deryugin) [inline src/cmds/testing/fb_draw_image/fb_draw_image.c]: I think this comment is redundant (name of the variable is pretty self-explanatory)

File: src/cmds/testing/fb_draw_image/fb_draw_image.c [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		printf("Error while getting file header\n");
+		close(fd);
+		return -1;
+	}
+	close(fd);
+
+	/* work with framebuffer */
+	fb_info = fb_lookup(0);
+
+	printf("%" PRId32 "x%" PRId32 ", %" PRId32 "bpp\n",fb_info->var.xres, fb_info->var.yres, fb_info->var.bits_per_pixel);
```
Comment (Deryugin) [inline src/cmds/testing/fb_draw_image/fb_draw_image.c]: `"bpp\n",fb_info` -> `"bpp\n", fb_info`

File: src/cmds/testing/fb_draw_image/fb_draw_image.c [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+			if (bmp_load(argv[1], &bmp_data) != 0) {
+				printf("Error : can't load bmp file.\n");
+				return -1;
+			}
+
+			if (get_fbp(fbp, fb_info, bmp_data.image, bmp_data.width, bmp_data.height) != 0) {
+				return -1;
+			}
+
+			free(bmp_data.image);
```
Comment (Deryugin) [inline src/cmds/testing/fb_draw_image/fb_draw_image.c]: It's better to call `malloc()` symmetrically `free()`. In this case: if you allocate memory in `lib/bmp` then you should free it in the same module. You can do it with function like `bmp_deinit`/`bmp_unload`/etc

File: src/cmds/testing/fb_draw_image/fb_draw_image.c [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+			break;
+
+		default:
+			printf("This format not support\n");
```
Comment (Deryugin) [inline src/cmds/testing/fb_draw_image/fb_draw_image.c]: `This format is not supported`

File: src/cmds/testing/fb_draw_image/fb_draw_image.c (hunk 1 above)
Comment (FilippChubukov) [reply src/cmds/testing/fb_draw_image/fb_draw_image.c]: remove

File: src/cmds/testing/fb_draw_image/fb_draw_image.c (hunk 2 above)
Comment (FilippChubukov) [reply src/cmds/testing/fb_draw_image/fb_draw_image.c]: done

File: src/cmds/testing/fb_draw_image/fb_draw_image.c (hunk 3 above)
Comment (FilippChubukov) [reply src/cmds/testing/fb_draw_image/fb_draw_image.c]: add new fun bmp_unload

File: src/cmds/testing/fb_draw_image/fb_draw_image.c (hunk 4 above)
Comment (FilippChubukov) [reply src/cmds/testing/fb_draw_image/fb_draw_image.c]: fix

**Pull request merged**